src/components/Post/Post.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { TypePost } from '../../types';
+import comments from '../../api/comments';
+import users from '../../api/users';
+import { CommentList } from '../CommentList';
+import { User } from '../User';
+
+import './Post.scss';
+
+export const Post = (props) => {
```

Comment:
desctructure props here

```suggestion
export const Post = ({ body, etc }) => {
```